aws/environments/EXAMPLE.yml

```diff
@@ -3,17 +3,16 @@
 # The following variables should be put into a new yml file in this directory.
 # Any variables used by ansible roles may also be set here.
 
-region: us-west-2
+region: us-east-1
```

Comment:
In general, we'd like all AWS resources to be in us-east-1 region by default - I'm hoping people clone this file and use us-east-1 going forward :)